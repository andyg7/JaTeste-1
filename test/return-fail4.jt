func int main()
{

	return 0;
}

func int do_something(int a, int b, int c, int d)
{
	int i;
	return a + b + c + d;
	i = i + 1;
}
