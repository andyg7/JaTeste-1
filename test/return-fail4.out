Scanned
Parsed
Fatal error: exception Exceptions.InvalidReturnType("Can't have any code after return statement")
